plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -333,6 +333,21 @@ public void testInsert()
         throw new SkipException("TODO");
     }
 
+    @Test
+    public void testInsertIntoTableHavingRowUuid()
+    {
+        String tableName = "test_insert_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM region WITH NO DATA", 0);
```

Comment:
We could use `TestTable` which would add the suffix and could drop the table once the test is completed ?